scripts/setup/bootstrap.sh

```diff
@@ -113,7 +113,7 @@ _bootstrap_or_activate() {
     local _PIGWEED_CIPD_JSON="$_CHIP_ROOT/third_party/pigweed/repo/pw_env_setup/py/pw_env_setup/cipd_setup/pigweed.json"
     mkdir -p "$_PW_ACTUAL_ENVIRONMENT_ROOT"
```

Comment:
Blocking comment as I want to validate a few assumptions before this can be merged